pom.xml

```diff
@@ -39,157 +58,131 @@
       <archive>https://mail-archives.apache.org/mod_mbox/logging-commits/</archive>
     </mailingList>
   </mailingLists>
-  <licenses>
-    <license>
-      <name>Apache License, Version 2.0</name>
-      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
-  <scm>
-    <connection>scm:git:https://github.com/apache/logging-log4net</connection>
-    <developerConnection>scm:git:https://github.com/apache/logging-log4net</developerConnection>
-    <url>https://github.com/apache/logging-log4net</url>
-  </scm>
-  <organization>
-    <name>Apache Software Foundation</name>
-    <url>https://www.apache.org</url>
-  </organization>
+
+  <ciManagement>
+    <system>GitHub Actions</system>
+    <url>https://github.com/apache/logging-log4net/actions</url>
+  </ciManagement>
+
+  <properties>
+
+    <!-- project version -->
+    <revision>0.10.0-SNAPSHOT</revision>
+
+    <!-- `project.build.outputTimestamp` is required to be present for reproducible builds.
+         We actually inherit one from the `org.apache:apache` through our parent `org.apache.logging:logging-parent`.
+         Though inheriting this property has two undesired consequences:
+         1. `artifact:compare` dumps an `ERROR` log stating that this `property should not be inherited but defined` (apache/logging-parent#50)
+         2. This value is employed in various places while creating the distribution
+         To mitigate these, we define a *dummy* value here and let the CI replace it during a release.
+         Hence, *DO NOT MANUALLY EDIT THIS VALUE*! -->
+    <project.build.outputTimestamp>2024-05-09T11:40:45Z</project.build.outputTimestamp>
+
+    <!-- disable `maven-site-plugin`-->
+    <maven.site.skip>true</maven.site.skip>
+    <maven.site.deploy.skip>true</maven.site.deploy.skip>
+    <spotbugs.skip>true</spotbugs.skip>
+  </properties>
+
+  <!-- `dependencyManagement` must only contain `log4j-tools` modules and nothing else!
```

Comment:
I think you can remove the entire `dependencyManagement` element.

Reply:
check